lib/change_stream.js

```diff
@@ -267,15 +267,18 @@ class ChangeStreamCursor extends Cursor {
   }
 
   get resumeOptions() {
-    if (this.resumeToken) {
-      return { resumeAfter: this.resumeToken };
+    const result = {};
+    for (const optionName of CURSOR_OPTIONS) {
+      if (this.options[optionName]) result[optionName] = this.options[optionName];
     }
 
-    if (this.startAtOperationTime && maxWireVersion(this.server) >= 7) {
-      return { startAtOperationTime: this.startAtOperationTime };
+    if (this.resumeToken) {
```

Comment:
I think in this case we need to make sure to delete any pre-existing `startAtOperationTime`, `resumeAfter`, or `startAfter` values for both of these cases:

```js
const RESUME_OPTIONS = ;
if (this.resumeToken || this.startAtOperationTime) {
  ['resumeAfter', 'startAfter', 'startAtOperationTime'].forEach(key => delete result[key]);
  if (this.resumeToken) {
    result.resumeAfter = this.resumeToken;
  } else if (this.startAtOperationTime && maxWireVersion(this.server) >= 7) {
    result.startAtOperationTime = this.startAtOperationTime;
  }
}
```

Reply:
done